gateway/gateway.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright the Hyperledger Fabric contributors. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+option go_package = "github.com/hyperledger/fabric-protos-go/gateway";
+option java_multiple_files = true;
+option java_package = "org.hyperledger.fabric.protos.gateway";
+option java_outer_classname = "GatewayProto";
+
+package protos;
+
+import "peer/proposal.proto";
+import "common/common.proto";
+
+service Gateway {
+    rpc Endorse(ProposedTransaction) returns (PreparedTransaction) {}
```

Comment:
A previous [effort](https://jira.hyperledger.org/browse/FAB-15306) changed all of the service definitions to use the simple semi-colon terminator instead of the empty brace block when we don't have annotations. Can we do that here as well to be consistent?

I see that another snuck in for snapshot so I'll go change that.